src/Google/Ads/GoogleAds/Lib/V1/GoogleAdsUnaryCallLogger.php

```diff
@@ -27,13 +28,14 @@
 final class GoogleAdsUnaryCallLogger
 {
     use ArrayTrait;
-    use GoogleAdsLoggerTrait;
 
     private $logger;
     private $logLevel;
     private $endpoint;
     private $logMessageFormatter;
     private $context;
+    private static $logLevelList;
```

Comment:
Could you use "private const" and change this to CAPITAL LETTERs instead?
It was not possible in PHP 5, but it should be fine now.

Reply:
This is not possible even in PHP7.1, because const fields cannot be initialized with an expression, they have to be initialized with a literal that can be evaluated at compile time.

Thus, initializing them as we do here with `array_keys(Logger::getLevels())` will throw a fatal error.